GetService/handler.ts

```diff
@@ -40,13 +44,25 @@ type IGetServiceHandler = (
   serviceId: ServiceId
 ) => Promise<IGetServiceHandlerRet>;
 
+export function serviceAvailableNotificationChannels(
+  retrievedService: RetrievedService
+): ReadonlyArray<NotificationChannel> {
+  if (retrievedService.requireSecureChannels) {
```

Comment:
Another way would have been to have two distinct service types (ie. `SecureService` and `InsecureService`) where 

```
const SecureService = t.interface({ requireSecureChannels: t.literal(true), 
   availableotificationChannels: t.readOnlyArray(typeof NotificationChannelEnum.WEBHOOK));
```
so you cannot have inconsistent objects (ie. requireSecureChannels=true and available_notification_channels=email,...)

Reply:
requireSecureChannels is an attribute of the **Service** type, this attribute is mapped as array of available notification channels in the **ServicePublic** type. We decided to hide the concept of secure channels and return to the app only a list of available (the user can switch on/off) channels.